src/buckets/components/BucketCardMeta.test.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import {renderWithRedux} from 'src/mockState'
+import BucketCardMeta from './BucketCardMeta'
+import {OwnBucket} from '../../types'
+import {SchemaType} from '../../client'
+
+const setup = (
+  schemaType: SchemaType,
+  type: 'user' | 'system' = 'system',
+  id: string
+) => {
+  const bucket: OwnBucket = {
+    id,
+    type,
+    schemaType,
+    readableRetention: '30 days',
+    name: 'testBucket',
+    retentionRules: [],
+    labels: [],
+  }
+  return renderWithRedux(<BucketCardMeta bucket={bucket} />)
+}
+
+describe(
+  'bucket meta data card, ' + 'testing that the schema type shows up properly',
```

Comment:
is this string concatenation supposed to be here?

Reply:
pretty-quick added that.

Reply:
...i fixed the whitespace.  after the build passes will merge up.